chatmaild/src/chatmaild/filtermail.py

```diff
@@ -230,6 +250,61 @@ def check_DATA(self, envelope):
         for recipient in envelope.rcpt_tos:
             if recipient_matches_passthrough(recipient, passthrough_recipients):
                 continue
+            p = self.config.mailboxes_dir.joinpath(recipient, "inclear")
```

Comment:
what is this "inclear"??? sorry if I overlooked the explanation somewhere :thinking:

Reply:
no, it's a very good point! this is left-over and needs to change, but i first need to figure out why no test complained.

Reply:
fixed in https://github.com/chatmail/server/pull/538/commits/e74e942ed2131fc5fcf1ab2f2f3e4f3e4d6ef843